ringpop-admin.js

```diff
@@ -38,6 +38,8 @@ function main() {
         .command('status', 'Status of members in ring')
         .command('partitions', 'Show partition information of a ring')
         .command('top', 'General membership information')
+        .command('reap', 'Remove the faulty nodes from this nodes membership')
+        .command('heal', 'Start a partition heal coordinated by the target node')
```

Comment:
I think we need to add a comment that the user probably needs to execute the heal a couple of times (reincarnation phase + merge phase). In the readme also. And also when the command runs. I don't think you can read that from the response (which is a shame) but something like: "heal algorithm consists of two phases, please run this command again after ~10 seconds until the partition is healed".